scripts/serie_update.py

```diff
@@ -142,6 +149,24 @@ def os_cmd(self, cmd, cwd=None, shell=False):
             stderr=self.std_dest,
             cwd=cwd,
         )
+
+    def rename_conf_template(self, path):
+        """renames hal_conf_template.h to hal_conf.h ...
+        Args:
+            path: path where to apply the files processing
+        """
+        hal_conf_template = [
+            f for f in path.iterdir() if "hal_conf_template.h" in f.name
+        ][0]
```

Comment:
why iterate to obtain a known file name?

Reply:
Inheritance of initial scripts. Reworked.